pex/scie/science.py

```diff
@@ -247,33 +265,49 @@ def _path_science():
 
 def _ensure_science(
     url_fetcher=None,  # type: Optional[URLFetcher]
-    science_binary_url=None,  # type: Optional[str]
+    science_binary=None,  # type: Optional[Union[File, Url]]
     env=ENV,  # type: Variables
 ):
     # type: (...) -> str
 
+    if isinstance(science_binary, File):
```

Comment:
This new handling for `--scie-science-bnary` when it points to a local science copy was used to test against science PyPy support before it released.